experiment/measurer/measure_manager.py

```diff
@@ -150,7 +150,7 @@ def save_snapshots():
         if not snapshots:
             return
 
-        db_utils.bulk_save(snapshots)
+        db_utils.add_all(snapshots)
```

Comment:
This is needed since Snapshot has relationship to crashes and we need to add them both. bulk_save does not do relationships.

Reply:
ACK